### ISSUE TITLE
matplotlib in installation deck

### ISSUE THREAD

**Issue by quan3010** [UBC-MDS/UBC-MDS.github.io#200]
@joelostblom @chendaniely 

Can we add matplotlib as part of the installation deck in the future?

**Comment by joelostblom** [comment]
Yes sure, which lecture do they need it in and were there any particular errors they ran into that we should watch out for if adding it to the installation instructions?

And then my usual question: Would it makes sense to have them use Altair instead of MPL here or is what they are plotting not a data frame?

**Comment by quan3010** [comment]
It's currently missing in the installation deck so they just run into issues module matplotlib not found. I am using it just for 1 exercise now so it's not a huge problem. However, we should have at least one viz library installed in Python by default so it'll allow them to run the visualization code in lab/lecture notes.

I have no preference for Altair or matplotlib, but matplotlib is more commonly used in other MDS ML courses as far as I know.

**Comment by chendaniely** [comment]
Did not realize mpl was not a dependency already. Pandas plotting uses mpl but i guess it's not required.
